Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -802,6 +794,14 @@ - (UIView *)loadViewForMediaGallery
     self.unloadCellContentBlock = ^{
         [galleryView unloadMedia];
     };
+    for (UIView *itemView in galleryView.itemViews) {
+        OWSBubbleShapeView *strokeView = [[OWSBubbleShapeView alloc] initDraw];
+        strokeView.strokeColor = [UIColor colorWithWhite:0.5f alpha:0.4f];
+        strokeView.strokeThickness = 1.f;
+        [itemView addSubview:strokeView];
+        [self.bubbleView addPartnerView:strokeView];
+        [self.viewConstraints addObjectsFromArray:[strokeView ows_autoPinToSuperviewEdges]];
+    }
```

Comment:
I'm not sure there's going to be a convenient way to use inner shadows on these views.